src/transformers/tokenization_transfo_xl.py

```diff
@@ -165,7 +169,8 @@ def __init__(
         lower_case=False,
         delimiter=None,
         vocab_file=None,
-        pretrained_vocab_file=None,
+        pretrained_vocab_file: str = None,
```

Comment:
This breaks backwards compatibility for people that have saved a torch tokenizer file locally and continue passing it as `pretrained_vocab_file` no? Is it maybe possible to just have `pretrained_vocab_file` as before and check of it is a torch or pickle file?

Reply:
Minor suggestion: in Ludwig I adopted the convention that when the variable ends with `_file` it is an opened `File`, and when it ends with `_fp` it is a string containing a file path. That may help resolving the ambiguity.

But Patrick is right, changing from one type to another right now may break backwards compatibility.

Reply:
Doesn't break compatibility anymore!